downloader/src/funTest/kotlin/DownloaderTest.kt

```diff
@@ -106,8 +106,8 @@ class DownloaderTest : StringSpec() {
                 Downloader().download(pkg, outputDir)
             }
 
-            exception.cause shouldNotBe null
-            exception.cause!!.message shouldBe "Source artifact does not match expected SHA-1 hash " +
+            exception.suppressed.size shouldBe 3
+            exception.suppressed.get(1)!!.message shouldBe "Source artifact does not match expected SHA-1 hash " +
```

Comment:
If we expect three suppressed exceptions we should verify that all of them are correct, not only the second.

Reply:
I intentionally did not do that in the first as I considered it out of scope of this commit and of this test as well. Anyhow, it seems good to have it covered, addressed in a separate commit.